Monika After Story/game/script-topics.rpy

```diff
@@ -4560,6 +4560,31 @@ label monika_ageinf:
     m "But I don't believe that my life can be summed up in a number."
     m 5a "So I'll always love you for all eternity, [player]. Just remember that."
     return
+    
+    init 5 python:
```

Comment:
Remove indentation before the `init` block. `init` python blocks **cannot** have any indentation